README.md

```diff
@@ -90,6 +91,161 @@ This WordPress project is intended to provide:
   define( 'SCRIPT_DEBUG', true );
   ```
 
+## Automated Testing
```

Comment:
What I would want to see here is how to run tests: all of them, a single test, or a group (if applicable). A simplified version of usage section.